example-get-started-experiments/code/.github/workflows/deploy-model-sagemaker.yml

```diff
@@ -26,6 +26,7 @@ jobs:
   deploy-model:
     needs: parse
     if: "${{ needs.parse.outputs.event == 'assignment' }}"
+    environment: cloud
```

Comment:
In #271, we changed the role from `AWS_SAGEMAKER_ROLE` to `AWS_SANDBOX_ROLE` and dropped `environment: cloud`. I'm taking a guess here that the `AWS_SAGEMAKER_ROLE` exists and has the correct value and was failing because it's only scoped to the `cloud` environment, but I need @shcheklein to confirm.